src/api.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+
+const RECORD_FIELDS_TO_CLEAN = ["_status", "last_modified"];
+
+export function cleanRecord(record, exludeFields=RECORD_FIELDS_TO_CLEAN) {
+  return Object.keys(record).reduce((acc, key) => {
+    if (exludeFields.indexOf(key) === -1)
+      acc[key] = record[key];
+    return acc;
+  }, {});
+};
+
+const DEFAULT_REQUEST_HEADERS = {
+  "Accept":       "application/json",
+  "Content-Type": "application/json",
+};
+
+export default class Api {
+  constructor(remote, options={}) {
+    this.remote = remote;
+    this._options = options;
+    try {
+      this.version = "v" + remote.match(/\/v(\d+)$/)[1];
+    } catch (err) {
+      throw new Error("The remote URL must contain the version: " + remote);
```

Comment:
Might be worth to check somewhere that remote is a string, otherwise this error message could be misleading